src/plugins/inputs/index.js

```diff
@@ -35,6 +35,7 @@ const userInputs = [
 	"rejoin",
 	"topic",
 	"whois",
+	"mute",
```

Comment:
Should unmute be present too?

Reply:
unmute is in `src/plugins/inputs/mute.js`, and this array is referring to the filenames. I can split them into two files if that's preferred, I don't really care.

Reply:
single file is preferred I think, we have that for others too